training/01_Introduction/03_Configs_Roles.adoc

```diff
@@ -0,0 +1,5 @@
+== Configs
+
+A config is simply a lab, workshop, or other deployment that has been defined and encapsulated in a series of playbooks, variable files, and templates in a config directory. This allows Ansible Agnostic Deployer to deploy it simply and repeatedly on one or more Cloud Platforms.
+
+More Specifically, configs build infrastructure in a cloud provider. Meaning that we will expect a set of resources to result from the execution of the config. Usually, these resources are defined in a jinja template, normally located in a /files or /templates directory in the config's hosting directory.
```

Comment:
@NonyNo3lle Just nit-picking because this is your PR.

Please reformat Asciidoc to have one sentence per line rather than a whole paragraph on a single line.

Also, do you mean to it introduce a `03_Configs_Roles.adoc` when there is already a `03_Infrastructure_Deployment`?

Reply:
yes, the idea is this will be slide 03 from Introduction module (training/01_Introduction), the 03_Infra_Deployment is in the Getting_Started module (02)

Reply:
Looking into format in a minute

Reply:
I think it's fine now, tho one sentence is kinda long. Let me know if it looks good.